index.rst

```diff
@@ -1,16 +1,24 @@
 OMERO guide
 ===========
 
+The OMERO guide offers step-by-step instructions on how to use components of the OMERO platform.
+To learn more about OMERO, we recommend that you first visit `https://www.openmicroscopy.org/omero/ <https://www.openmicroscopy.org/omero/>`_
+
 The OMERO guide is divided into several parts:
 
 :doc:`core_features` covers how to get started, import, manage data and run server-side OMERO scripts.
 
-:doc:`add_ons` includes instructions for installing Web, Insight and Command Line Interface plugins
+:doc:`web_addons` includes instructions for installing Web plugins
 and how to use those extensions points e.g. OMERO.figure, OMERO.parade.
 
+:doc:`add_ons` includes instructions for installing Insight and Command Line Interface plugins
+and how to use those extension points.
+
 :doc:`api_usage` allows clients to be written in Java, Python, R, C++ or MATLAB. This section includes installation instructions and an exhaustive list of examples.
 
-:doc:`external_tools` introduces how to analyze data using third party tools e.g. Fiji, CellProfiler, ilastik. 
+:doc:`external_tools` introduces how to analyze data using third party tools e.g. Fiji, CellProfiler, ilastik.
+
+:doc:`example` combines an example step-by-step workflow from import to figure creation.
```

Comment:
I think this is the nicest entry-point for anyone who's not sure what to look at. But it's quite buried low-down on this page. Maybe it could be moved after the first paragraph e.g. under a "Getting started" subheading, since it's kinda higher level than "The OMERO guide is divided into several parts"?